tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -19,6 +19,7 @@
 <dai-copier-be.conf>
 <dai-copier-gain-module-copier-capture.conf>
 <gain-capture.conf>
+<gain-copier-capture.conf>
```

Comment:
Commit message: please don't use 'reference' for the description of the input stream. This is an overloaded term and not linked at all in this case with the 'AEC reference'.

Reply:
@plbossart  sure will take it off.